enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationService.java

```diff
@@ -106,7 +103,7 @@ public class IdentityVerificationService {
      * @param auditService Audit service.
      */
     @Autowired
-    public IdentityVerificationService(
+    IdentityVerificationService(
```

Comment:
Let's keep the service _public_ because of API, but hide the constructor little bit, because `IdentityVerificationPrecompleteGuard` is not public on purpose.

Reply:
Ok.